modules/FindOVR.cmake

```diff
@@ -0,0 +1,25 @@
+#
+# Find OVR libraries
+#
+#
+
+set(OVR_INCLUDE_DIR ${CMAKE_INSTALL_PREFIX}/include/)
+message(STATUS "OVR_INCLUDE_DIR = " ${OVR_INCLUDE_DIR})
+
+if (MSVC)
+	find_library(OVR_LIBRARY NAMES "libOVR.lib")
+elseif (MINGW)
+	# linking against the MSVC dll with MinGW does not work directly. Instead, you need to
+	# link against a specific version. This will cause problems with newer oculus runtimes,
+	# though. (FIXME!)
+	find_library(OVR_LIBRARY NAMES "LibOVRRT32_0_6.dll" HINTS "C:/Windows/SysWOW64/")
+endif()
+
+set(OVR_LIBRARIES
+	${OVR_LIBRARY}
+)
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(OVR OVR_LIBRARY OVR_LIBRARIES OVR_INCLUDE_DIR)
+
+message(STATUS "OVR_FOUND = " ${OVR_FOUND})
```

Comment:
These two status messages shouldn't be needed (`find_package_handle_standard_args()` prints a message on success/failure anyway).